drivers/mspi/mspi_nrfe.c

```diff
@@ -104,10 +96,7 @@ static void ep_recv(const void *data, size_t len, void *priv)
 }
 
 static struct ipc_ept_cfg ep_cfg = {
-	.cb = {
-		.bound = ep_bound,
-		.received = ep_recv,
-	},
+	.cb = {.bound = ep_bound, .received = ep_recv},
```

Comment:
```suggestion
	.cb = {.bound = ep_bound, .received = ipc_recv_clbk},
```
Then `ep_recv` is unnecessary.